packages/editor-ui/src/mixins/genericHelpers.ts

```diff
@@ -29,6 +29,9 @@ export const genericHelpers = defineComponent({
 		readOnlyEnv(): boolean {
 			return this.sourceControlStore.preferences.branchReadOnly;
 		},
+		isLoading(): boolean {
```

Comment:
hope we can get rid of these mixins soon.. maybe this is better to keep in nodeview?

Reply:
Yeah me too. Makes sense since it's currently only used there, will update